sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample11_RecognizeIdDocuments.md

```diff
@@ -0,0 +1,210 @@
+# Recognize Id Documents
+
+This sample demonstrates how to recognize and extract common fields from Id documents, using a pre-trained model. For a suggested approach to extracting information from ID documents, see [strongly-typing a recognized form][strongly_typing_a_recognized_form].
```

Comment:
And if we prefer to keep them as `Id` instead of `ID`, we should rename this one `ID documents` to `Id documents` for consistency.